app/src/androidTest/java/org/mozilla/fenix/syncintegration/SyncIntegrationTest.kt

```diff
@@ -113,30 +132,48 @@ class SyncIntegrationTest {
         }
     }
 
-    /* These tests will be running in the future
-    // once the test above runs successfully and
-    // the environment is stable
-
-    // History item Fenix -> Desktop
+    // Bookmark item Fenix -> Desktop
     @Test
     fun checkBookmarkFromDeviceTest() {
-        tapInToolBar()
-        typeInToolBar()
-        seeBookmark()
-        mDevice.pressBack()
-        signInFxSync()
+        val defaultWebPage = "example.com".toUri()!!
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(defaultWebPage) {
+        }.openThreeDotMenu {
+            verifyAddBookmarkButton()
+            clickAddBookmarkButton()
+        }
+        browserScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openTurnOnSyncMenu {
+            useEmailInsteadButton()
+            typeEmail()
+            tapOnContinueButton()
+            typePassword()
+            sleep(TestAssetHelper.waitingTimeShort)
```

Comment:
Do you have to hide the save login prompt once you sign-in?

Reply:
No, I have not found an issue with having to hide it.  Once the tap on Sign In button is done,  the android part of the test finishes so that prompt does not interfere with the rest of this or other tests...